data/pathfinder/paizo/roleplaying_game/core_essentials/ce_abilities_race.lst

```diff
@@ -225,15 +225,16 @@ CATEGORY=Internal|Base Size ~ Colossal.MOD
-Pincers		CATEGORY:Internal		TYPE:NaturalAttack.Secondary	DEFINE:NaturalAttacks|0	DEFINE:PincersAttacks|0		BONUS:VAR|PincersAttacks|2	BONUS:VAR|NaturalAttacks|1			BONUS:WEAPONPROF=Pincers|TOHIT|-5	BONUS:WEAPONPROF=Pincers|DAMAGE|-(STR/2)
+Pincers		CATEGORY:Internal		TYPE:NaturalAttack.Secondary	DEFINE:NaturalAttacks|0	DEFINE:PincersAttacks|0		BONUS:VAR|PincersAttacks|2	BONUS:VAR|NaturalAttacks|PincersAttacks			BONUS:WEAPONPROF=Pincers|TOHIT|-5	BONUS:WEAPONPROF=Pincers|DAMAGE|-(STR/2)
 Tail Slap		CATEGORY:Internal		TYPE:NaturalAttack.Secondary	DEFINE:NaturalAttacks|0	DEFINE:TailSlapAttacks|0	BONUS:VAR|TailSlapAttacks|1	BONUS:VAR|NaturalAttacks|1			BONUS:WEAPONPROF=Tail Slap|TOHIT|-5	BONUS:WEAPONPROF=Tail Slap|DAMAGE|-(STR/2)
 Claw			CATEGORY:Internal		TYPE:NaturalAttack.Primary	DEFINE:NaturalAttacks|0	DEFINE:ClawAttacks|0		BONUS:VAR|ClawAttacks|2	BONUS:VAR|NaturalAttacks|ClawAttacks		BONUS:WEAPONPROF=Claw|DAMAGE|MAX(STR/2,0)|PREVAREQ:NaturalAttacks,1
 Slam			CATEGORY:Internal		TYPE:NaturalAttack.Primary	DEFINE:NaturalAttacks|0	DEFINE:SlamAttacks|0		BONUS:VAR|SlamAttacks|1	BONUS:VAR|NaturalAttacks|1				BONUS:WEAPONPROF=Slam|DAMAGE|MAX(STR/2,0)|PREVAREQ:NaturalAttacks,1
 Sting			CATEGORY:Internal		TYPE:NaturalAttack.Primary	DEFINE:NaturalAttacks|0	DEFINE:StingAttacks|0		BONUS:VAR|StingAttacks|1	BONUS:VAR|NaturalAttacks|1			BONUS:WEAPONPROF=Sting|DAMAGE|MAX(STR/2,0)|PREVAREQ:NaturalAttacks,1
 Talons		CATEGORY:Internal		TYPE:NaturalAttack.Primary	DEFINE:NaturalAttacks|0	DEFINE:TalonsAttacks|0		BONUS:VAR|TalonsAttacks|2	BONUS:VAR|NaturalAttacks|TalonsAttacks	BONUS:WEAPONPROF=Talons|DAMAGE|MAX(STR/2,0)|PREVAREQ:NaturalAttacks,1
-Hoof			CATEGORY:Internal		TYPE:NaturalAttack.Secondary	DEFINE:NaturalAttacks|0	DEFINE:HoofAttacks|0		BONUS:VAR|HoofAttacks|2	BONUS:VAR|NaturalAttacks|1				BONUS:WEAPONPROF=Hoof|TOHIT|-5	BONUS:WEAPONPROF=Hoof|DAMAGE|-(STR/2)
-Tentacle		CATEGORY:Internal		TYPE:NaturalAttack.Secondary	DEFINE:NaturalAttacks|0	DEFINE:TentacleAttacks|0	BONUS:VAR|TentacleAttacks|2	BONUS:VAR|NaturalAttacks|1			BONUS:WEAPONPROF=Tentacle|TOHIT|-5	BONUS:WEAPONPROF=Tentacle|DAMAGE|-(STR/2)
-Wing			CATEGORY:Internal		TYPE:NaturalAttack.Secondary	DEFINE:NaturalAttacks|0	DEFINE:WingAttacks|0		BONUS:VAR|WingAttacks|2	BONUS:VAR|NaturalAttacks|1				BONUS:WEAPONPROF=Wing|TOHIT|-5	BONUS:WEAPONPROF=Wing|DAMAGE|-(STR/2)
+Hoof			CATEGORY:Internal		TYPE:NaturalAttack.Secondary	DEFINE:NaturalAttacks|0	DEFINE:HoofAttacks|0		BONUS:VAR|HoofAttacks|2	BONUS:VAR|NaturalAttacks|HoofAttacks				BONUS:WEAPONPROF=Hoof|TOHIT|-5	BONUS:WEAPONPROF=Hoof|DAMAGE|-(STR/2)
+HoofPony	CATEGORY:Internal		TYPE:NaturalAttack.Primary	DEFINE:NaturalAttacks|0	DEFINE:HoofAttacks|0		BONUS:VAR|HoofAttacks|2	BONUS:VAR|NaturalAttacks|HoofAttacks				BONUS:WEAPONPROF=Hoof|DAMAGE|MAX(STR/2,0)|PREVAREQ:NaturalAttacks,1
```

Comment:
BONUS:WEAPONPROF=Hoof|DAMAGE|MAX(STR/2,0)|PREVAREQ:NaturalAttacks,1 (Also, unless there has been a change to the var, don't equine companions get two attacks with their hoofs?  Attack 2 hooves (1d3); which means the code never triggers...Since NaturalAttacks is set to 2.